libs/ledger-live-common/src/families/evm/__snapshots__/bridge.integration.test.ts.snap

```diff
@@ -1,3263 +0,0 @@
-// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Why is this whole file removed ?

Reply:
I ran `pnpm common jest --update-snapshot` and this was removed

Reply:
Well that's part of my more general comment, I don't get why you update a snapshot in a PR supposed to just add a abandon seed, it shouldn't be related.

Reply:
Actually it's related to this action that had an incorrect snapshot: https://github.com/LedgerHQ/ledger-live/actions/runs/3360932223/jobs/5570716730#step:9:5122